### PR TITLE
remove sleep time

### PR DESCRIPTION
**Pull request by sumitnigamcs**
@tivv @rmstar @ksahilCF  Removed sleep time, please review

### CODE REVIEW

File: src/main/java/io/cdap/e2e/pages/actions/CdfStudioActions.java [hunk 1]
```diff
@@ -16,64 +16,68 @@
...
-      PageFactory.initElements(SeleniumDriver.getDriver(), io.cdap.e2e.pages.locators.CdfStudioLocators.class);
+      PageFactory.initElements(SeleniumDriver.getDriver(), CdfStudioLocators.class);
   }
 
   public static void selectGCS() throws InterruptedException {
-    Thread.sleep(3000);
-    io.cdap.e2e.pages.locators.CdfStudioLocators.gcsBucket.click();
+    WebElement element = CdfStudioLocators.gcsBucket;
+    SeleniumHelper.waitElementIsVisible(element, 1);
+    element.click();
```
Comment (tivv) [inline src/main/java/io/cdap/e2e/pages/actions/CdfStudioActions.java]: I see that wait and click is an often combination. Would you mind making a utility function with "1" as a default parameter

File: src/main/java/io/cdap/e2e/utils/SeleniumHelper.java [hunk 2]
```diff
@@ -93,7 +93,7 @@ public static void sendKeys(WebElement element, String keys) {
     element.sendKeys(keys);
   }
 
-  public static boolean waitElementIsVisible(WebElement element) throws InterruptedException {
+  public static boolean waitElementIsVisible(WebElement element, long timeoutInSec) throws InterruptedException {
     WebDriverWait wait = new WebDriverWait(SeleniumDriver.getDriver(), 60);
```
Comment (tivv) [inline src/main/java/io/cdap/e2e/utils/SeleniumHelper.java]: I don't see you use the timeout

File: src/main/java/io/cdap/e2e/pages/actions/CdfStudioActions.java (hunk 1 above)
Comment (sumitnigamcs) [reply src/main/java/io/cdap/e2e/pages/actions/CdfStudioActions.java]: created

File: src/main/java/io/cdap/e2e/utils/SeleniumHelper.java (hunk 2 above)
Comment (sumitnigamcs) [reply src/main/java/io/cdap/e2e/utils/SeleniumHelper.java]: corrected

File: src/main/java/io/cdap/e2e/utils/SeleniumHelper.java [hunk 3]
```diff
@@ -93,10 +93,18 @@ public static void sendKeys(WebElement element, String keys) {
...
+    WebDriverWait wait = new WebDriverWait(SeleniumDriver.getDriver(), timeoutInSec);
     wait.until(ExpectedConditions.visibilityOf(element));
-    return true;
+  }
+
+  /*
+  Waits for visibility and clicks, with default value 1 unless provided.
+  timeOutInSec is an optional parameter
+   */
+  public static void waitAndClick(WebElement element, long ...timeOutInSec) throws InterruptedException {
```
Comment (rmstar) [inline src/main/java/io/cdap/e2e/utils/SeleniumHelper.java]: you don't need varargs here. 

void waitAndClick(WebElement element, long timeOutInSec) {}

void waitAndClick(WebElement element) {
  waitAndClick(element, DEFAULT_TIMEOUT);
}

Comment (sumitnigamcs) [reply src/main/java/io/cdap/e2e/utils/SeleniumHelper.java]: done

File: src/main/java/io/cdap/e2e/pages/actions/CdfLogActions.java [hunk 4]
```diff
@@ -18,29 +18,29 @@
 
 import io.cdap.e2e.pages.locators.CdfLogLocators;
 import io.cdap.e2e.utils.SeleniumDriver;
+import org.openqa.selenium.WebElement;
 import org.openqa.selenium.support.PageFactory;
 
+import static io.cdap.e2e.utils.SeleniumHelper.waitAndClick;
```
Comment (rmstar) [inline src/main/java/io/cdap/e2e/pages/actions/CdfLogActions.java]: prefer not to use static import

Comment (sumitnigamcs) [reply src/main/java/io/cdap/e2e/pages/actions/CdfLogActions.java]: corrected

File: src/main/java/io/cdap/e2e/pages/actions/CdfBigQueryPropertiesActions.java [hunk 5]
```diff
@@ -16,10 +16,16 @@
...
 
+import io.cdap.e2e.pages.locators.CdfBigQueryPropertiesLocators;
 import io.cdap.e2e.pages.locators.CdfStudioLocators;
 import io.cdap.e2e.utils.SeleniumDriver;
+import io.cdap.e2e.utils.SeleniumHelper;
 import org.openqa.selenium.support.PageFactory;
 
+import java.io.IOException;
+
+import static io.cdap.e2e.utils.SeleniumHelper.readParameters;
```
Comment (rmstar) [inline src/main/java/io/cdap/e2e/pages/actions/CdfBigQueryPropertiesActions.java]: remove this static import as well

Comment (sumitnigamcs) [reply src/main/java/io/cdap/e2e/pages/actions/CdfBigQueryPropertiesActions.java]: removed

**Review by rmstar — APPROVED** [review]

**Pull request merged**